### DISCUSSION TITLE
Does this render engine have motion blur?

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#283] [Q&A] [answered]
I'm willing to learn Malt since it has pretty much anything that I can't do in EEVEE, but I'm wondering if this comes with a motion blur feature especially motion vectors that can be exported as AOV?

**Comment by pragma37** [comment, marked as answer]
Not yet. You will have to resort to Cycles for now.

**Reply by Psi25** [reply]
Ah, I see. Thanks for the reply.
